lib/include/react_controller/baxterChain.h

```diff
@@ -54,7 +54,7 @@ class BaxterChain : public KDL::Chain
      *
      * return:     true if successful, false if error
      */
-    bool JntToCart(const KDL::JntArray& _q_in, KDL::Frame& _p_out, int _segNr=-1);
+    bool JntToCart(const KDL::JntArray& _q_in, KDL::Frame& _p_out, int seg_nr=-1);
```

Comment:
Yes I believe so. What do you mean? I just changed its name to have similar naming to `JntToJac`